hatchet/graphframe.py

```diff
@@ -313,40 +313,62 @@ def to_hdf(self, filename, key="hatchet_graphframe", **kwargs):
         HDF5Writer(filename).write(self, key=key, **kwargs)
 
     def copy(self):
-        """Return a shallow copy of the graphframe.
+        """Return a partially shallow copy of the graphframe.
```

Comment:
Thoughts on the wording here? I'm hesitant to just say `shallow` since it's more of a custom copy but most of the behavior is "shallow-ish" in comparison to the `deepcopy`.